frontend/src/components/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -96,66 +96,78 @@ class CrossingListItem extends React.Component {
-    this.setState({ selectedDuration: null });
+    this.setState({ 
+      selectedStatus: statusConstants.CAUTION,
+      notes: '',
+      selectedReason: this.props.reasons.find(reason => reason.statusId === statusConstants.CAUTION).id,
+      selectedDuration: null
+    });
   };
   closedClicked = () => {
-    this.setState({ selectedStatus: statusConstants.CLOSED });
-    this.setState({ notes: '' });
-    this.setState({ selectedReason: this.props.reasons.find(reason => reason.statusId === statusConstants.CLOSED).id });
-    this.setState({ selectedDuration: null });
+    this.setState({ 
+      selectedStatus: statusConstants.CLOSED,
+      notes: '',
+      selectedReason: this.props.reasons.find(reason => reason.statusId === statusConstants.CLOSED).id,
+      selectedDuration: null
+    });
   };
   longtermClicked = () => {
-    this.setState({ selectedStatus: statusConstants.LONGTERM });
-    this.setState({ notes: '' });
-    this.setState({ selectedReason: this.props.reasons.find(reason => reason.statusId === statusConstants.LONGTERM).id });
-    this.setState({ selectedDuration: this.props.durations[0].id });
+    this.setState({ 
+      selectedStatus: statusConstants.LONGTERM,
+      notes: '',
+      selectedReason: this.props.reasons.find(reason => reason.statusId === statusConstants.LONGTERM).id,
+      selectedDuration: this.props.durations[0].id
+    });
   };
 
   reasonChanged = (e) => { this.setState({ selectedReason: e.target.value }) };
   durationChanged = (e) => { this.setState({ selectedDuration: e.target.value }) };
   notesChanged = (e) => { this.setState({ notes: e.target.value }) };
 
   cancelClicked = () => {
-    this.setState({ selectedStatus: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusId });
-    this.setState({ selectedReason: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusReasonId });
-    this.setState({ selectedDuration: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusDurationId });
-    this.setState({ notes: this.props.crossing.statusUpdateByLatestStatusUpdateId.notes });
+    this.setState({ 
+      selectedStatus: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusId,
+      selectedReason: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusReasonId,
+      selectedDuration: this.props.crossing.statusUpdateByLatestStatusUpdateId.statusDurationId,
+      notes: this.props.crossing.statusUpdateByLatestStatusUpdateId.notes 
+    });
   };
 
+  isDirty() {
+    // Temporary fix for storybook
+    if(this.props.dirty) return true;
+
+    const savedStatus = this.props.crossing.statusUpdateByLatestStatusUpdateId.statusId;
+    const savedReason = this.props.crossing.statusUpdateByLatestStatusUpdateId.statusReasonId;
+    const savedDuration = this.props.crossing.statusUpdateByLatestStatusUpdateId.statusDurationId;
+    const savedNotes = this.props.crossing.statusUpdateByLatestStatusUpdateId.notes;
```

Comment:
I like the ES6 Object destructuring syntax because it can reduce these 4 lines with a lot of repeating info into one. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

Ex: 
```js
const { statusId, statusReasonId, statusDurationId, notes } = this.props.crossing.statusUpdateByLatestStatusUpdateId;
```

Reply:
I actually did that refactor in my search PR, completely agree